README.md

```diff
@@ -3,6 +3,11 @@
 ## Overview
 The edX learning platform's frontend for enterprise learners.
 
+## Some dev notes
+This section can be removed once the items are addressed:
+
+-[ ] remove dep  "eslint-plugin-import": "^2.22.1" once frontend-build upgrade is done https://github.com/edx/frontend-build/pull/136
```

Comment:
Looks like the frontend-build upgrade is done! 😄

Reply:
yup I did not gett it released inlast job, trying again with https://travis-ci.com/github/edx/frontend-build/builds/202213889